config/environments/development.rb

```diff
@@ -53,4 +53,18 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.default_url_options = { host: "localhost", port: "3000" }
+
+   config.action_mailer.delivery_method = :smtp
+    config.action_mailer.smtp_settings = {
+    address: "smtp.gmail.com",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.